src/containers/SocketContainer.js

```diff
@@ -0,0 +1,84 @@
+import React, { Component, createElement} from 'react';
+import { socketResponse } from '../actions/SocketAction'
+import createSocket, { Sockette } from "sockette-component";
+import { connect } from 'react-redux';
+
+const Socket = createSocket({
+  Component,
+  createElement
+});
+
+class SocketContainer extends Component {
+    // Socket internal state
+    state = {
```

Comment:
get rid of local state please!

Reply:
reduced local state as much as we can. Must keep state.socket